test/integration/test-amp-ad-3p.js

```diff
@@ -200,6 +200,8 @@ describe('amp-ad 3P', () => {
           );
         });
         lastIO = null;
+
+        // Ad is fully visible
```

Comment:
also document that the ad is fully visible in the initial place
(so that the position observer behavior is covered)

Reply:
done